config/environments/production.rb

```diff
@@ -40,9 +40,11 @@
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   # config.force_ssl = true
 
-  # Use the lowest log level to ensure availability of diagnostic information
-  # when problems arise.
-  config.log_level = :debug
+  # Include generic and useful information about system operation, but avoid logging too much
+  # information to avoid inadvertent exposure of personally identifiable information (PII).
+  config.log_level = :warn
```

Comment:
@starswan What do you think of using `ENV.fetch("LOG_LEVEL") { :warn }` ?

Reply:
There doesn't appear to be a need. CCQ doesn't do it

Reply:
@starswan I "info" would log request parameters, which we don't want.

Reply:
It doesn't. That's what the switch to 'lograge' logging does.